sites/models.py

```diff
@@ -32,6 +32,13 @@ class Site(ClusterableModel):
         unique=True,
         help_text='Specify the domain name without the scheme, '
                   'e.g. "example.com" instead of "https://example.com"')
+    onion_address = models.URLField(
+        blank=True,
+        help_text=('The Onion address for this site. This field takes '
+                   'precedence over the Onion-Location header from scan '
+                   'results for determining if a site is available over Onion '
+                   'services.'),
+    )
```

Comment:
I think we will need to add this in the panel for allowing adding/editing onion addresses